examples/examples/event_error_details.rs

```diff
@@ -77,7 +77,7 @@ async fn main() {
 			let string_error = format!("{:?}", e);
 			// We expect a TokenError::FundsUnavailable error. See :
```

Comment:
I think you can remove this comment as well. The FundsUnavailable is clearly visible in the code itself.